src/app/containers/CpsAssetMediaPlayer/__snapshots__/index.test.jsx.snap

```diff
@@ -1,21 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`MediaPlayer render the amp player 1`] = `
-<html>
```

Comment:
if we're no longer using shouldMatchSnapshot then we should maybe have an assertion to check that all this stuff still gets rendered

Reply:
I have managed to fix this and revert back to using `shouldMatchSnapshot`